python/ray/_private/parameter.py

```diff
@@ -447,15 +447,6 @@ def build_error(resource, alternative):
         if self.redirect_output is not None:
             raise DeprecationWarning("The redirect_output argument is deprecated.")
 
-        # Parse the numpy version.
```

Comment:
This is not needed any more -- numpy 1.16 only supported Python up to 3.7, which we are deprecating (see https://pypi.org/project/numpy/1.16.0/#files)